test/e2e/tests/metamask-responsive-ui.spec.js

```diff
@@ -178,10 +178,7 @@ describe('Metamask Responsive UI', function () {
 
         await driver.fill('#password', 'correct horse battery staple');
         await driver.fill('#confirm-password', 'correct horse battery staple');
-        await driver.clickElement({
-          text: enLocaleMessages.restore.message,
-          tag: 'button',
-        });
+        await driver.press('#confirm-password', driver.Key.ENTER);
```

Comment:
one thing I don't really like about this is it only tests keyboard input. it might be nice to continue testing both? but idk if there's really an effective way. you could introduce some randomness, but part of testing is making these reproducible so :shrug: 

idea:
```suggestion
        if (Math.floor(Math.random() * 2)) {
          await driver.clickElement({
            text: enLocaleMessages.restore.message,
            tag: 'button',
          });
        } else {
          await driver.press('#confirm-password', driver.Key.ENTER);
        }
```

Reply:
🤔 Not sure if we should do this. Seems like this thread has some interesting perspectives: https://sqa.stackexchange.com/questions/33444/how-do-you-design-ui-automated-tests-to-pick-alternative-steps-randomly-in-a-flo

Reply:
Oh my goodness, please no randomness :weary:. Not unless you're trying to tank team morale. The e2e tests fail intermittently enough as-is.

Reply:
Testing both cases would make sense though. Maybe not here in the "responsive layout" test, but somewhere.

Reply:
hey, it was just an idea, but I agree it's a bad one! I didn't commit the suggestion for a reason. I think we can just leave it as testing with `<enter>` (the way it is now). I'm confident that as long as the button is `submit` it will function as desired.